css/material.css

```diff
@@ -1,3 +1,7 @@
+#sim{
+
```

Comment:
That's where we change the background colour if we want, I was playing with that but left the new gradient in the end.